apps/web/src/dialogs/command-palette/commands.ts

```diff
@@ -72,8 +72,14 @@ export const commandActions = {
   command: (command: Command) => getCommandById(command.id)?.action(command),
   note: (command: Command, options?: { openInNewTab?: boolean }) =>
     useEditorStore.getState().openSession(command.id, options),
-  notebook: (command: Command) => navigate(`/notebooks/${command.id}`),
-  tag: (command: Command) => navigate(`/tags/${command.id}`),
+  notebook: (command: Command) => {
+    useAppStore.getState().setNavigationTab("notebooks");
+    navigate(`/notebooks/${command.id}`);
```

Comment:
Instead of doing this in 2 steps everywhere, we could just listen to the `onNavigate` event (we already listen to it in `navigation-menu`) and change the tab based on the route. It would greatly simplify things.